glue/qt/widgets/image_widget.py

```diff
@@ -74,6 +76,9 @@ def _setup_widgets(self):
         self.ui.slice_layout.addWidget(self.ui.slice)
         self._tweak_geometry()
 
+        self.ui.aspectCombo.addItem("Square Pixels", userData='equal')
```

Comment:
I've had some issues with the behavior of userData across different Qt bindings, though I don't remember the details. It's fine to leave this for now, but we should keep this in mind in case we see bug reports in the future